src/Microsoft.DotNet.Cli.Utils/Command.cs

```diff
@@ -93,7 +117,7 @@ private static string ResolveExecutableFromPath(string executable, ref string ar
             return executable;
         }
 
-        private static string ResolveExecutablePathFromProject(string executable, NuGetFramework framework)
+        private static string ResolveExecutablePathFromProject(string executable, NuGetFramework framework, ref CommandResolutionStrategy commandResolution)
```

Comment:
ref is a pretty bad code smell here. This should get fixed by PackagedCommands PR currently under review, though...